examples/ProtoBuffer/protobuffer.ino

```diff
@@ -0,0 +1,64 @@
+#include <TheThingsNetwork.h>
+#include <TheThingsMessage.h>
+
+// Set your AppEUI and AppKey
+const byte appEui[8] = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 };
+const byte appKey[16] = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00
+, 0x00 };
+
+#define loraSerial Serial1
+#define debugSerial Serial
+
+TheThingsNetwork ttn(loraSerial, debugSerial, /* TTN_FP_EU868 or TTN_FP_US915 */);
+
+sensorData data = api_Measurement_init_default;
+
+void setup()
+{
+  pinMode(2, INPUT);
+
+  loraSerial.begin(57600);
+  debugSerial.begin(9600);
+
+  // Wait a maximum of 10s for Serial Monitor
+  while (!debugSerial && millis() < 10000);
+
+  debugSerial.println("-- STATUS");
+  ttn.showStatus();
+
+  debugSerial.println("-- JOIN");
+  ttn.join(appEui, appKey);
+
+  //Chose message you want to send (send the message by writing true)
```

Comment:
I thought choosing the message you want to send had it place in the ``` setup() ``` part. But I guess we could put it in the ``` loop() ``` .